gcp/api/server.py

```diff
@@ -733,6 +741,12 @@ def do_query(query: osv_service_v1_pb2.Query,
         'version specified in params and purl query',
     )
 
+  # Hack to work around ubuntu having extremely large individual entries
+  if (ecosystem.startswith("Ubuntu") or
```

Comment:
I think only `linux` related packages would reach the limit. Should we only apply this to packages start with `linux`?

Reply:
Applied to only get linux packages.